### ISSUE TITLE
Graphing support: add nomnoml

### ISSUE THREAD

**Issue by maulwuff** [lunet-io/markdig#148]
Would it be possible to add further graphing support to the existing mermaid extension?
Mermaid is really weak on class diagrams, so I started looking around and found http://www.nomnoml.com/ from https://github.com/skanaar/nomnoml
which has really nice syntax and produces nice diagrams.
Maybe something like

      ```nomnoml
      ```
To insert a nomnoml block?

**Comment by maulwuff** [comment]
to add a little more input:

nomnoml can draw to a canvas, or output the diagram in svg:
this is a minimal example showing both types:

```html
<html>
    <head>
        <script src="zepto.min.js"></script>
        <script src="lodash.min.js"></script>
        <script src="dagre.min.js"></script>
        <script src="nomnoml.js"></script>
    </head>
    <body>
        <canvas id="target-canvas"></canvas>
        <div id="nomnomldiv"></div>
        <script>
            var canvas = document.getElementById('target-canvas');
            var source = '[nomnoml] is -> [awesome]';
            nomnoml.draw(canvas, source);
            
            var divid = document.getElementById('nomnomldiv');
            divid.innerHTML = nomnoml.renderSvg(source);
        </script>
    </body>
</html>
```

it would be enough to let markdig output a div just like it is done with mermaid:
```
        <div class="nomnoml">
            [class irtzr]->[class vbr] 
            [df458 ]->[nvjht877fg]
        </div>
```

this could be parsed afterwards like this:
```
        <script>
            function htmlDecode(input)
            {
              var doc = new DOMParser().parseFromString(input, "text/html");
              return doc.documentElement.textContent;
            }
            var graphs = document.getElementsByClassName('nomnoml');
            for (var i = 0; i < graphs.length; i++) {
                graphs[i].innerHTML = nomnoml.renderSvg(htmlDecode(graphs[i].innerHTML));
            }            
        </script>
```

**Comment by xoofx** [comment]
This is a one line add in markdig (see [DiagramExtension.cs#L27](https://github.com/lunet-io/markdig/blob/42472085a62dfb86e229f1cbb16e24aead6a9741/src/Markdig/Extensions/Diagrams/DiagramExtension.cs#L27)), so PR welcome if you want this.  😉

**Comment by maulwuff** [comment]
I can only provide patches. thanks to .net core 2 I am not able to compile or do anything :-/

[nomnomlSupport.zip](https://github.com/lunet-io/markdig/files/1390442/nomnomlSupport.zip)

**Comment by maulwuff** [comment]
finally managed to create a pull request: #149 
only took me 6 hours :rofl:

**Comment by xoofx** [comment]
Fixed by #149
